### ISSUE TITLE
[A11y] Explore Collections - Grid - High contrast failures

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1888]
### Operating System

Windows

### Browser

Edge

### Affected page

Explore Collections

### Admin user?

Yes

### Issue Description

High contrast mode loses images:

![image](https://user-images.githubusercontent.com/2157974/129251795-a03ec1fd-b217-4891-a385-94f40c5308b7.png)

### Suggested Fix

We should avoid using single-color images like this one: https://staging.oregondigital.org/assets/collection-a38b932554788aa578debf2319e8c4ba8a7db06b3ba57ecda1391a548a4b6e0a.png

If you edit that, you'll find that it's black and transparent.  That means on any dark background it's invisible.  It should probably have white borders around the black, or else not be transparent.

### Can anybody QA or does this need an expert?

Anybody

### QA Steps

Switch to high-contrast mode and verify the page is usable.

**Comment by jechols** [comment]
Another issue I missed last time: the selected view (table vs. grid) is essentially hidden, and the "down arrow" on the results per page button gets really weird:

![image](https://user-images.githubusercontent.com/2157974/129274863-a65ba1a3-4f36-49dd-9027-5140adf172dc.png)

**Comment by jechols** [comment]
The explore collections page is currently showing me no images for the collections, so it's hard to test... I'll check again tomorrow.

**Comment by jechols** [comment]
Images are back, which is great, but unfortunately they still disappear in high contrast mode because they're still black with a transparent background:

![image](https://user-images.githubusercontent.com/2157974/146613717-b0702673-0075-4c52-9e6b-fea18345c9d8.png)

**Comment by azle** [comment]
Updated 1/6/2024 FIXED

![IMG_8515](https://github.com/user-attachments/assets/c0900630-e8bd-4f48-bb63-87d025d338e2)
